### PR TITLE
Ank apply exits on not scheduled

### PR DESCRIPTION
**Pull request by krucod3**
Issues: #241 

Apply exits on not scheduled + added hint that the waiting for the target states can be interrupted.
Additionally added some convenience scripts and aliases.

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [ ] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: ank/src/cli_commands/wait_list.rs [hunk 1]
```diff
@@ -115,3 +127,214 @@ impl<T: WaitListDisplayTrait> WaitList<T> {
...
+//////////////////////////////////////////////////////////////////////////////
+//                 ########  #######    #########  #########                //
+//                    ##     ##        ##             ##                    //
+//                    ##     #####     #########      ##                    //
+//                    ##     ##                ##     ##                    //
+//                    ##     #######   #########      ##                    //
+//////////////////////////////////////////////////////////////////////////////
+
+#[cfg(any(feature = "test_utils", test))]
+fn generate_test_wait_list(
```
Comment (lingnoi) [inline ank/src/cli_commands/wait_list.rs:140]: make it public to share the test functionality  within the 'test_utils' context, or remove the 'feature' config item if it is not intended to be public
```suggestion
#[cfg(any(feature = "test_utils", test))]
pub fn generate_test_wait_list(
```

File: ank/src/cli_commands.rs [hunk 2]
```diff
@@ -1041,6 +1041,8 @@ impl CliCommands {
         if changed_workloads.is_empty() {
             output!("No workloads to update");
             return Ok(());
+        } else {
+            output!("Applied successfully the manifest(s). Waiting for target workload states (to skip, interrupt with Ctrl+C).\n");
```
[the remark this answers is not in the document]
Comment (krucod3) [reply ank/src/cli_commands.rs:1045]: How about:
```suggestion
            output!("Applied successfully the manifest(s). Waiting for target workload states (press Ctrl+C to interrupt waiting).\n");
```

File: ank/src/cli_commands/wait_list.rs (hunk 1 above)
Comment (krucod3) [reply ank/src/cli_commands/wait_list.rs:140]: Goo catch 👍, you are right, either public or not exposed to the outside ```suggestion
#[cfg(test)]
fn generate_test_wait_list(
```

**Review by maturar — APPROVED** [review]
LGTM :+1:

**Review by lingnoi — APPROVED** [review]
LGTM 👍

**Pull request merged**